asmcomp/cmm.mli

```diff
@@ -134,10 +134,10 @@ and expression =
   | Clet of Ident.t * expression * expression
   | Cassign of Ident.t * expression
   | Ctuple of expression list
-  | Cop of operation * expression list
+  | Cop of operation * expression list * Debuginfo.t
   | Csequence of expression * expression
   | Cifthenelse of expression * expression * expression
-  | Cswitch of expression * int array * expression array
+  | Cswitch of expression * int array * expression array * Debuginfo.t
```

Comment:
> which correspond to actual instructions in the output

Cassign (resp. Cexit) is likely to result in a "mov" (resp. "jmp") instruction in the result, no?  Cloop will also produce instructions which can be attributed to that specific loop.  Both Cassign and Cloop can also be traced back to a specific piece of the source program (less so for Cexit).